python/F3DPythonBindings.cxx

```diff
@@ -92,7 +92,7 @@ PYBIND11_MODULE(f3d, module)
     .export_values();
 
   loader
-    .def("addFiles", &f3d::loader::addFiles, "Add files to the loader", py::arg("list of files"))
+    .def("addFiles", &f3d::loader::addFiles, "Add files to the loader", py::arg("list of files"), py::arg("recursive") = false)
```

Comment:
This file will need more changes, but it can be done later

Reply:
yes